procsim/biomass/level1_product_generator.py

```diff
@@ -322,75 +323,75 @@ def generate_output(self):
             self._hdrs.append(self._hdr)
 
         for hdr in self._hdrs:
-            self._generate_level1_stacked_product(hdr)
+            self._hdr = hdr
```

Comment:
I don't like the use of the member variable as function parameter here. In the future, we should create the 'Product' class (as discussed earlier).

Reply:
I have added a TODO for now (7f02b4e).